tests/e2e/tests/solution-server/solution-server-analysis-validation.test.ts

```diff
@@ -0,0 +1,140 @@
+import { expect, test } from '../../fixtures/test-repo-fixture';
```

Comment:
since the test is under solution-server folder. 
Test can be renamed to "analysis-validation.test.ts"